package.json

```diff
@@ -40,16 +40,21 @@
   },
   "devDependencies": {
     "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "^7.16.0",
+    "@babel/eslint-parser": "^7.16.3",
     "@playwright/test": "^1.16.3",
     "@storybook/addon-actions": "^6.3.12",
     "@storybook/addon-essentials": "^6.3.12",
     "@storybook/addon-links": "^6.3.12",
     "@storybook/addon-storysource": "^6.3.12",
     "@storybook/react": "^6.3.12",
+    "@testing-library/dom": "^8.11.1",
+    "@testing-library/jest-dom": "^5.15.0",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.0.2",
```

Comment:
Please remove these dependencies. They were here in an older version of the project but are not used anymore.
After this is done, make sure you have npm 7 or 8, then delete the package-lock.json, run `npm install`, and commit the new package-lock.json.

Reply:
i deleted those lines and run npm install => i got  `55 vulnerabilities (3 low, 15 moderate, 35 high, 2 critical)`
my npm version is 8.1.0

Reply:
and i updated all packages before install (so this is why package-lock) changed